### PR TITLE
Add russian localization

### PR DESCRIPTION
**Pull request by f1nality**

### CODE REVIEW

File: post_office/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
+"%100>=11 && n%100<=14)? 2 : 3);\n"
+#: post_office/admin.py:81 post_office/models.py:200
+msgid "Render template in alternative language"
+msgstr "Отправить письмо на другом языке"
```
Comment (yprez) [inline post_office/locale/ru/LC_MESSAGES/django.po]: @f1nality "Отправить письмо" ("sending an email") isn't quite the same as "render a template", I wonder if the original meaning is preserved...

File: post_office/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+msgid "Emails"
+msgstr "Письма"
+
+#: post_office/models.py:167
+msgid "Email address"
+msgstr "Email адрес"
+
+#: post_office/models.py:170
+msgid "Exception type"
+msgstr "Тип исключения"
```
Comment (yprez) [inline post_office/locale/ru/LC_MESSAGES/django.po]: "exception" means "error" in this context. Wouldn't "тип ошибки" be better here?

**Comment by yprez** [comment]
Otherwise looks good to me...

**Comment by yprez** [comment]
@selwin on another note, isn't it better not to store .mo files in source control?

Those are binary (generated from .po files by `manage.py compile_messages`), and might be different per system...

File: post_office/locale/ru/LC_MESSAGES/django.po (hunk 1 above)
Comment (f1nality) [inline post_office/locale/ru/LC_MESSAGES/django.po]: @yprez there is no good equivalent for a word "render" in russian, so I replaced with russian translation for "Send mail in alternative language".

File: post_office/locale/ru/LC_MESSAGES/django.po (hunk 2 above)
Comment (f1nality) [inline post_office/locale/ru/LC_MESSAGES/django.po]: Yes, for "error" "ошибка" is better. I though python exception was supposed.

File: post_office/locale/ru/LC_MESSAGES/django.po (hunk 1 above)
Comment (yprez) [inline post_office/locale/ru/LC_MESSAGES/django.po]: Agreed

**Comment by selwin** [comment]
@yprez I don't Django's translation features so I'm not that familiar with it so I'll defer to you on this one :)

**Comment by yprez** [comment]
@selwin I'll make a different PR removing `.mo` files for all translations (and a more detailed explanation why).

No reason to hold this one because of this, it just follows the convention used in the repository...

**Comment by selwin** [comment]
@yprez alright, thanks!

**Pull request merged**